### PR TITLE
Feature user can upvote a question

### PR DESCRIPTION
**Pull request by jwharrow**
currently allows user to vote more than once.
currenrtly separate forms for upvote and downvote. could combine forms to single form and make it work with CSS.

### CODE REVIEW

File: app/controllers/votes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+post '/questions/:id/vote' do
+  vote = Vote.new(value: params[:value])
+  vote.user = current_user
+  vote.save
+
+  @question = Question.find_by(id: params[:id])
+  @question.votes << vote
+  erb :"questions/show"
```
Comment (cdfuller) [inline app/controllers/votes_controller.rb:8]: Currently broken. This view also requires `@answers` to render. I believe redirecting to '/questions/:id' would be a better option here.

File: joe_notes.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+joe_notes.md
```
Comment (cdfuller) [inline joe_notes.md]: This file isn't needed.

**Review by cdfuller — CHANGES_REQUESTED** [review]
Vote validation needs work. Need to insure user can only vote once.

**Comment by cdfuller** [comment]
Currently working on this.

File: app/controllers/votes_controller.rb (hunk 1 above)
Comment (jwharrow) [reply app/controllers/votes_controller.rb:8]: agree. made changes.

**Pull request closed without merging**